daml-foundations/daml-ghc/test-src/DA/Test/ShakeIdeClient.hs

```diff
@@ -431,7 +432,10 @@ goToDefinitionTests mbScenarioService = Tasty.testGroup "Go to definition tests"
                 ]
             setFilesOfInterest [foo]
             expectNoErrors
+            -- This actually ends up pointing to "concat".
```

Comment:
@rohanjr Maybe something for you to look at as part of the new desugarer (I haven’t really looked into it for now since it seems very likely that the new desugarer will change this).